src/main/java/appeng/client/gui/widgets/GuiImgButton.java

```diff
@@ -769,6 +770,19 @@ public GuiImgButton(final int x, final int y, final Enum idx, final Enum val) {
                     ButtonToolTips.PriorityCardMode,
                     ButtonToolTips.PriorityCardMode_Dec);
 
+            this.registerApp(
+                    64,
+                    Settings.INTERFACE_TERMINAL_SECTION_ORDER,
+                    StringOrder.NATURAL,
+                    ButtonToolTips.StringOrder,
+                    ButtonToolTips.StringOrderNatural);
+            this.registerApp(
+                    65,
```

Comment:
I suggest changing these to 16 or 49 (for natural sort order) and 64 (for alpha-numeracal). Icon 65 is used for count and looks odd here.

Reply:
to clarify first since the names are a little confusing
NATURAL = Java natural comparator (a1, a11, a2)
ALPHANUM = new comparator that sorts naturally (a1,a2,a11)

I think icon 64 is a good fit for NATURAL since it sorts by alphabet. 
I changed the icon for ALPHANUM to 16, I wouldn't use 49 for that in case anyone ever wants to make a button that changes the sort order from ascending to descending.